components/molecules/DevCard/dev-card.tsx

```diff
@@ -186,10 +190,10 @@ export default function DevCard(props: DevCardProps) {
                 height={50}
                 className="absolute rounded-full border-[1.5px] border-gray-300 border-opacity-40 -left-1"
               />
-              <p className="pl-10">{props.devstats?.login}</p>
+              <p className="pl-10 truncate items-center">{props.devstats?.login}</p>
               <p className="-pl-4 justfy-self-end text-sm flex gap-0.5">
```

Comment:
Center the text. Also it looked like it was never self justifying to the end.

```suggestion
              <p className="-pl-4 justify-self-end items-center text-sm flex gap-0.5">
```